### PR TITLE
ENH: Add jsonlogic binary

### PR DESCRIPTION
**Pull request by mplanchard**
Add a binary to use jsonlogic from the commandline. Input data can be
provided as an argument or via stdin. Multiple calls can be chained
together, parsing the output of the previous call as data.

### CODE REVIEW

File: src/bin.rs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+use serde_json;
+use serde_json::Value;
+
+use jsonlogic_rs;
+
+fn configure_args<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
+    app.version(env!("CARGO_PKG_VERSION"))
+        .author("Matthew Planchard <msplanchard@gmail.com>")
+        .about(
+            "Parse JSON data with a JsonLogic rule.\n\
```
Comment (mkatychev) [inline src/bin.rs:15]: would it be easier to read if we make about a `r#` multiline string?

File: README.md [hunk 2]
```diff
@@ -109,14 +116,55 @@ res = jsonlogic_rs.apply(
...
+``` raw
+Parse JSON data with a JsonLogic rule.
+
+When no <data> or <data> is -, read from stdin.
+
+The result is written to stdout as JSON, so multiple calls
+can be chained together if desired.
+
+USAGE:
+    jsonlogic <logic> [data]
```
Comment (mkatychev) [inline README.md:138]:
```suggestion
    jsonlogic <logic> [<data>]
```

**Review by mkatychev — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mplanchard) [reply README.md:138]: this is auto-generated by `clap`. I think it uses `<name>` for required arguments and `[name]` for optional ones. I can see about changing it

File: src/bin.rs (hunk 1 above)
Comment (mplanchard) [reply src/bin.rs:15]: I don't like how they have to go all the way over to the left side of the screen 🤷‍♂️

File: README.md (hunk 2 above)
Comment (mkatychev) [reply README.md:138]: I see, that would technically not be docopt compliant unless [`data` was a literal input](http://docopt.org/) but I think that's no big deal if it's autogened

**Pull request merged**